blog/_posts/2017-12-11-Thanks-for-diversifying-tech-with-us.md

```diff
@@ -0,0 +1,50 @@
+twitter: ines_opcoelho
+categories:
+- news
+---
+
+For the past five years, every (north hemisphere) summer, RGSoC has been offering the opportunity for women (transgender and cisgender) and non-binary people from all over the world to work on an Open Source project. This wouldn't be possible without the help and love of our wonderful community. 
+
+![Together we are changing the world!](/img/blog/2017/2017-12-12-change-the-world.jpg)
+<div class="image-credits">Our 20 teams - the lives we touched on the 2017 edition of RGSoC (Image: Inês Coelho)</div>
+
+### Our sponsors and donors
+
+Our program is based on a crowdfunding and sponsorship campaign that runs all year round, whose money is used to finance our teams. This year we were able to fund 16 out of our 20 participating teams of two! 
+
+We can't thank enough all the companies that [sponsored](https://railsgirlssummerofcode.org/sponsors/) RGSoC: we wouldn't be here without you! A special thank you goes to our amazing partners, [Travis CI](https://travis-ci.com/) and [GitHub](https://github.com/), our platinum sponsor [Nokia](https://www.nokia.com) and our gold sponsors, [Google Open Source](https://opensource.google.com/) and [Malwarebytes](https://www.malwarebytes.com/). 
+
+We are also extremely grateful to the 161 individuals who donated to our campaign. You are the best! 
+
+And, of course, a huge thank you to all the [conferences](https://railsgirlssummerofcode.org/students/conferences/#conferences) that welcomed our participants and donated tickets to our teams! You are awesome!
+
+### Our supporting elements
+
+Participating teams are supported throughout the summer by [dedicated volunteers](https://railsgirlssummerofcode.org/about/team). 
+
+There are no words to express our gratitude towards them all: the mentors who guided the work of our teams, the coaches that offered their time and expertise to help our teams, the coaching companies who hosted our teams during the summer, the supervisors who made sure that everything was running smoothly, the [trust committee](https://railsgirlssummerofcode.org/about/code-of-conduct/#contact) who offered a safe haven to whoever needed it. 
+
+You are the people who made RGSoC a reality! Thank you.
+
+### Our organizers
+
+Under the hood of RGSoC, there is a melting pot of people from different countries, cultures, backgrounds, beliefs and religions, united with the same purpose: to diversify tech! All year round, these volunteers are here preparing, planning, developing and working on the backend of RGSoC. 
+
+A huge shout-out to our 2017 team: Ana Sofia, Anika, Anna, Carsten, Inês, Laura, Lucas, Maria, Max, Ramon, Ulrike and Vaishali! 
+
+Your love and dedication are what makes RGSoC possible year after year.
+
+### Thank you all!
+
+From the bottom of our hearts: thank you! &#x2661;
+
+(click on the image for a surprise!)
+[![Thank you all](/img/blog/2017/2017-12-12-thankyou.gif)](https://railsgirlssummerofcode.org/thank-you-2017)
```

Comment:
aww, the thank you gif looks great!

Reply:
Thank you! <3 And it has the right size (I think) to be published on twitter if you want to use it to publicise the blog post.